tools/policy_master_permission_generation/validated_policy_templates.yaml

```
@@ -251,7 +251,8 @@ validated_policy_templates:
 -  "./cost/flexera/cco/budget_report_alerts/budget_report_alerts.pt"
 -  "./cost/flexera/cco/budget_v_actual_spend_report/budget_v_actual_spend_report.pt"
 -  "./cost/flexera/cco/budget_v_actual/monthly_budget_v_actual.pt"
--  "./cost/flexera/cco/cheaper_regions/cheaper_regions.pt"
+-  "./cost/flexera/cco/cbi_ingestion_aws_s3/cbi_ingestion_aws_s3.pt"
+-  "./cost/flexera/cco/cbi_ingestion_azure_blob/cbi_ingestion_azure_blob.pt"
```

```suggestion
- "./cost/flexera/cco/cheaper_regions/cheaper_regions.pt"
-  "./cost/flexera/cco/cbi_ingestion_aws_s3/cbi_ingestion_aws_s3.pt"
-  "./cost/flexera/cco/cbi_ingestion_azure_blob/cbi_ingestion_azure_blob.pt"
```
Was cheaper regions intentionally removed?

---

Nope! Fixed